client/model/ip_access_list.go

```diff
@@ -0,0 +1,46 @@
+package model
```

Comment:
in preparation of refactoring in #224, please move this to `workspace` package. `access` package might be good as well. (because it's `access` for `workspace`...)